src/CoreGraphics/CGContext.cs

```diff
@@ -1420,6 +1420,21 @@ public CGBitmapContext AsBitmapContext ()
 		{
 			return new CGBitmapContext (Handle, false);
 		}
+
+#if NET
+		[SupportedOSPlatform ("ios17.0")]
+		[SupportedOSPlatform ("maccatalyst17.0")]
+		[SupportedOSPlatform ("macos14.0")]
+		[SupportedOSPlatform ("tvos17.0")]
```

Comment:
why the diff in how we define the platform attrs for .net vs legacy? iirc these bindings that are within a framework directory go through one less pass through the generator, so assuming that has something to do with it?

Reply:
This is manually generated code, both need to be added. in the generator written code this is added. Look at the .g.cs files.